src/xdocs/checks.xml

```diff
@@ -401,32 +401,34 @@
               Checks that interface type parameter names conform to a specified pattern.</td>
           </tr>
           <tr>
-            <td><a href="config_javadoc.html#InvalidJavadocPosition">
+            <td><a href="checks/javadoc/invalidjavadocposition.html#InvalidJavadocPosition">
               InvalidJavadocPosition</a></td>
             <td>
               Checks that Javadocs are located at the correct position.</td>
           </tr>
           <tr>
-            <td><a href="config_javadoc.html#JavadocBlockTagLocation">
+            <td><a href="checks/javadoc/javadocblocktaglocation.html#JavadocBlockTagLocation">
               JavadocBlockTagLocation</a></td>
             <td>Checks that a javadoc block tag appears only at the beginning of a line,
               ignoring leading asterisks and white space.</td>
           </tr>
           <tr>
-            <td><a href="config_javadoc.html#JavadocContentLocation">
+            <td><a href="checks/javadoc/javadoccontentlocation.html#JavadocContentLocation">
               JavadocContentLocation</a></td>
             <td>
               Checks that the Javadoc content begins from the same position
               for all Javadoc comments in the project.
             </td>
           </tr>
           <tr>
-            <td><a href="config_javadoc.html#JavadocMethod">JavadocMethod</a></td>
+            <td><a href="checks/javadoc/javadocmethod.html#JavadocMethod">JavadocMethod</a></td>
             <td>Checks the Javadoc of a method or constructor.</td>
           </tr>
           <tr>
             <td>
-              <a href="config_javadoc.html#JavadocMissingLeadingAsterisk">JavadocMissingLeadingAsterisk</a>
+              <a href="checks/javadoc/javadocmissingleadingasterisk.html#JavadocMissingLeadingAsterisk">
+                JavadocMissingLeadingAsterisk
+              </a>
             </td>
             <td>
               Checks if the javadoc has
```

Comment:
Right below here, we lost link in description: https://checkstyle-diff-reports.s3.us-east-2.amazonaws.com/20eddb3_2023152032/checks/javadoc/index.html

Seems to be related to issue at https://github.com/checkstyle/checkstyle/pull/13258#discussion_r1236976318

I wonder how hard it would be to have a check that makes sure that all check descriptions end with `.`

Reply:
Fixed.